.github/workflows/install-ssf.yaml

```diff
@@ -0,0 +1,33 @@
+name: CI
+on:
+  - pull_request
+  - workflow_dispatch 
+jobs:
+  job1:
+    runs-on: ubuntu-latest
+    name: Test SSF Installation
+    steps:
+    - uses: actions/checkout@v2
+    - name: Start minikube
+      run: |
+        make setup-minikube
+    - name: Try the cluster !
+      run: kubectl get pods -A
+    - name: Deploy SSF to minikube
+      run: |
+        make setup-tekton-chains
+        make setup-kyverno
+    - name: Generate temp keys
+      run: |
+        make tekton-generate-keys
+    - name: Run test pipeline
+      run: |
+        export COSIGN_PUB=${PWD}/cosign.pub
```

Comment:
I think this can be skipped if the cosign verifies use `k8s://tekton-chains/signing-secrets`

Reply:
Makes sense to me